.github/actions/usd-install-dep/action.yml

```
@@ -8,17 +8,26 @@ inputs:
   version:
     description: 'Version of usd to build'
     required: true
+  openexr_version:
+    description: 'Version of openexr to build against'
+    required: true
 
 runs:
   using: "composite"
   steps:
 
+    - name: Check required inputs
+      shell: bash
+      run: |
+        [[ "${{ inputs.version }}" ]] || { echo "version input is empty" ; exit 1; }
+        [[ "${{ inputs.openexr_version }}" ]] || { echo "openexr_version input is empty" ; exit 1; }
+
     - name: Cache USD
       id: cache-usd
-      uses: actions/cache@v4
+      uses: actions/cache/restore@v4
       with:
         path: dependencies/usd_install
-        key: usd-${{inputs.version}}-${{runner.os}}-${{inputs.cpu}}-3
+        key: usd-${{inputs.version}}-${{inputs.openexr_version}}-${{runner.os}}-${{inputs.cpu}}-3
```

Reset key increment to 0

```suggestion
        key: usd-${{inputs.version}}-${{inputs.openexr_version}}-${{runner.os}}-${{inputs.cpu}}-0
```